src/main/java/org/usfirst/frc4904/robot/commands/FlywheelSpinUp.java

```diff
@@ -0,0 +1,26 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.subsystems.Flywheel;
+import org.usfirst.frc4904.standard.commands.motor.MotorVelocitySet;
+public class FlywheelSpinUp extends MotorVelocitySet {
+  public static final double DEFAULT_FLYWHEEL_SPEED = 0.0; // TODO: This is an untested value
+
+  /**
+   * Spin up the flywheel to a speed
+   * 
+   * @param flywheel The flywheel to manipulate
+   * @param speed    The speed to spin the flywheel up to
+   */
+  FlywheelSpinUp(Flywheel flywheel, double speed) {
+    super("FlywheelSpinUp", flywheel, speed);
+  }
+
+  /**
+   * Spin up the flywheel to the default speed
+   * 
+   * @param flywheel The flywheel to manipulate
+   */
+  FlywheelSpinUp(Flywheel flywheel) {
+    this(flywheel, DEFAULT_FLYWHEEL_SPEED);
```

Comment:
does this need to be done constantly like `motorSet`? I'm not sure but we should check

Reply:
I had to constantly set the setpoint when testing the VelocitySensorMotor for Blinky, so I assumed this would act the same... though I agree, we should test/check.